app/main.py

```diff
@@ -1,13 +1,18 @@
 def format_linter_error(error: dict) -> dict:
-    # write your code here
-    pass
+    return {"line": error["line_number"], "column": error["column_number"],
+            "message": error["text"], "name": error["code"],
+            "source": "flake8"}
 
 
 def format_single_linter_file(file_path: str, errors: list) -> dict:
-    # write your code here
-    pass
+    return {"errors": [format_linter_error(error)
+            for error in errors],
+            "path": file_path, "status": "passed"
+            if not errors else "failed"}
```

Comment:
Place each key on a new line to improve the readability, there should be no keys on the same line with `{` and `}`